code/modules/projectiles/projectile/special/rocket.dm

```diff
@@ -91,15 +91,15 @@
 
 /obj/item/projectile/bullet/a84mm_he/on_hit(atom/target, blocked=0)
 	..()
-	explosion(target, 0, 1, 2, 4)
+	explosion(target, 0, 0, 2, 4)
```

Comment:
Right, why is that a bad thing to give it heavy radius when most rockets you gave to Legion and NCR have devastation radius?